classes/PodsI18n.php

```diff
@@ -176,6 +176,8 @@ private static function default_strings() {
 
 			'Search'                                       => __( 'Search', 'pods' ),
 
+			'Search %s'                                    => __( 'Search %s', 'pods' ),
```

Comment:
:no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: bdf703eb98ab59064fb398fec3cf0076</sub>